README.md

```diff
@@ -79,26 +79,33 @@ should reside in any repo where you want to sync security alerts with Jira.
 Here is an example setup which runs this action every 6 hours.
```

Comment:
If we add the `workflow_dispatch ` part then we should probably also mention it here and perhaps why it is nice to include.

Reply:
Good point. I have added some text.